conbench/app/compare.py

```diff
@@ -285,10 +285,17 @@ class CompareRuns(Compare):
 
 
 rule(
-    "/compare/benchmarks/<compare_ids>/",
+    "/compare/benchmark-results/<compare_ids>/",
     view_func=CompareBenchmarkResults.as_view("compare-benchmark-results"),
     methods=["GET"],
 )
+# legacy route
+rule(
+    "/compare/benchmarks/<compare_ids>/",
+    endpoint="compare-benchmarks",
+    redirect_to="/compare/benchmark-results/<compare_ids>/",
```

Comment:
I was looking for reference docs for the `redirect_to` arg. It's not documented here: Not mentioned on this page: https://flask.palletsprojects.com/en/2.2.x/api/

That's a bit suspicious / a bit of a red flag.

Also no result here (as of today):
https://github.com/pallets/flask/issues?q=is%3Aissue+is%3Aopen+%22redirect_to%22+

It is only mentioned here: https://flask.palletsprojects.com/en/2.3.x/patterns/favicon/?highlight=redirect_to -- doesn't seem to be a first-class API citizen.

Specifically, I was wondering about the HTTP response status code that this is specified to emit (there are various different kinds of redirect responses).

I have previously decided to to not use a redirect, but to simply have two routes using the same handler. I think that's more pragmatic.

Reply:
> I was looking for reference docs for the `redirect_to` arg. 

The kwargs from [`add_url_rule()`](https://flask.palletsprojects.com/en/2.3.x/api/#flask.Flask.add_url_rule) that this `rule()` helper is calling get sent to a werkzeug [`Rule`](https://werkzeug.palletsprojects.com/en/2.3.x/routing/#werkzeug.routing.Rule) class, where it's documented. 

> Specifically, I was wondering about the HTTP response status code that this is specified to emit (there are various different kinds of redirect responses).

While sorting out the testing for this I discovered it's definitely emitting a 308

> I have previously decided to to not use a redirect, but to simply have two routes using the same handler. I think that's more pragmatic.

That's fine too if this becomes a pain in any way; I had that for a bit in a previous iteration.